.changeset/dull-hounds-remain.md

```diff
@@ -0,0 +1,5 @@
+---
+'@chainlink/ncfx-adapter': patch
```

Comment:
We don't need a changeset. There are no source code changes, which means it does not need to re-build a new image.